org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/ProcessParameterPanel.java

```diff
@@ -1178,6 +1205,25 @@ private void dynamicDisplay() {
 					LayoutUtils.addSclass("idempiere-mandatory-label", editor.getLabel());
 				}
 			}
+			// Handle Dynamic Display for Date Range Picker
+			if (((mField.getDisplayType() == DisplayType.Date) || (mField.getDisplayType() == DisplayType.DateTime))
+					&& mField2 != null
+					&& ((mField2.getDisplayType() == DisplayType.Date) || (mField2.getDisplayType() == DisplayType.DateTime))
+					&& MProcessPara.DATERANGEOPTION_TextAndRangePicker.equalsIgnoreCase(mField.getDateRangeOption())) {
+				DateRangeEditor dateRangeEditor = m_dateRangeEditors.get(i);
+				if(dateRangeEditor != null) {
+					dateRangeEditor.setVisible(editor.isVisible());
+					m_Rows.get(i).setVisible(editor.isVisible());
+					m_Rows.get(i).setAttribute(Group.GROUP_ROW_VISIBLE_KEY, editor.isVisible());
+					editor.setVisible(false, true);
+					if (mField.getVO().isRange) {
+						m_separators.get(i).setVisible(false);
+						m_wEditors2.get(i).setVisible(false ,true);
+					}
```

Comment:
I think 1218-1222 is probably redundant since if date range editor is use, editor and editor2 would have already been set invisible before this.